dvc/progress.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import print_function
 import sys
 import threading
+import logging
```

Comment:
I imported `logging` directly to prevent a circular dependency between `Logger` and `Progress`